resources/assets/components/PagingButtons/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class PagingButtons extends React.Component {
+
+  render() {
+    const prev = this.props.prev;
+    const next = this.props.next;
+    const prevCopy = this.props.prevCopy;
+    const nextCopy = this.props.nextCopy;
+
+    return (
+        <div className="container__block">
+			<a href={prev}>{prevCopy}</a>
+			<div className="next-page">
+				<a href={next}>{nextCopy}</a>
```

Comment:
This is totally not needed for this pull request, but we should think about loading these in on the client-side so we don't need to have a full page refresh here.